### ISSUE TITLE
This will not work on Linux/MacOS 

### ISSUE THREAD

**Issue by supertobi** [gregzaal/HDR-Merge#2]
https://github.com/gregzaal/HDR-Merge/blob/1d7208c3a0344546292112efd97752da9836180b/hdr_brackets.py#L264

This will not work on Linux/MacOS or any Windows, that is not installed in the standard directory. I think the best solution is to deliver your one sound file.

**Comment by gregzaal** [comment]
True, though the next line `if os.path.exists(sf):` means the script will continue if that file doesn't exist.

**Comment by ldo** [comment]
I think a better option on Linux would be to send a completion notification via D-Bus.
